src/BenchmarksDriver/Program.cs

```diff
@@ -401,5 +428,10 @@ private static void Log(string message, Reporter reporter)
             var time = DateTime.Now.ToString("hh:mm:ss.fff");
             reporter.WriteLine($"[{time}] {message}");
         }
+
+        private static string PP(string[] args)
```

Comment:
I'll change this to an extension method in `Benchmarks.ClientJob` so it can be shared across the Driver and Client.